shell/test/specs/demo-tests.js

```diff
@@ -426,11 +426,11 @@ describe('Arcs demos', function() {
     // restaurant.
     const restaurantSelectors = particleSelectors('root', ['#webtest-title']);
     waitForVisible(restaurantSelectors);
-    waitForSuggestion('Make a reservation');
+    waitForSuggestion('Make a reservation.');
     let restaurantNodes = pierceShadows(restaurantSelectors);
     console.log(`click: restaurantSelectors`);
     browser.elementIdClick(restaurantNodes.value[0].ELEMENT);
-    acceptSuggestion('Table for 2');
+    acceptSuggestion('Table for 2 available at 19:00.');
```

Comment:
The time can sometimes be 18:30 or so. Maybe just look for " available at"?

There is a case that no table is available and we should make sure to accept that as well. I'll have to look up the text for that.

Reply:
ah! the problem is that i need the `.` at the end of the sentence to distinguish this from the coalesced suggestion... maybe will have to add regex matching then.